product_docs/docs/pgd/5/index.mdx

```diff
@@ -40,6 +40,7 @@ be configured between different nodes, groups of nodes or all nodes by configuri
 
 | EDB Postgres <br/> Distributed | BDR <br/> Extension | pgLogical3 <br/> Extension | HARP | Commmunity <br/> PostgreSQL | EDB Postgres <br/> Extended Server | EDB Postgres <br/> Advanced Server | PGD <br/> CLI |
 | ------------------------------ | ------------------- | -------------------------- | ---- | --------------------------- | ---------------------------------- | ---------------------------------- | ------------- |
+| 5                              | 5                   | n/a                        | 2FIXME | 12, 13, 14, 15            | 12, 13, 14, 15                     | 12, 13, 14, 15                     | 1?            |
```

Comment:
The HARP thing needs changing, I see following options:
- we put n/a there
- we remove HARP altogether
- we replace HARP with Proxy and put HARP 2 for 4 and older and PGD-Proxy for 5

Reply:
oh and PGD CLI is v5 with PGD5